tests/e2e/helpers/keys.ts

```diff
@@ -1,11 +1,11 @@
+import { Chance } from 'chance';
 import { KeyTypesTexts } from './constants';
 
 export function getRandomKeyName(keyNameLength: number): string {
+    const chance = new Chance();
     let result = '';
-    const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
-    const charactersLength = characters.length;
     for (let i = 0; i < keyNameLength; i++) {
-        result += characters.charAt(Math.floor(Math.random() * charactersLength));
+        result += chance.character();
```

Comment:
It means you use the default, are `!@#$%^&*()` valid characters for your use cases?

Reply:
Yes, it will be better to use whole symbol range. This function is used for key name generation which can contain any symbols